lib/spectacles/version.rb

```diff
@@ -1,3 +1,3 @@
 module Spectacles
-  VERSION = "0.5.3"
+  VERSION = "0.5.4"
```

Comment:
Typically, it's not advised to bump versions in the same PR as a change because it can cause conflicts if two people bump the version, or the version is bumped in master before the PR is merged in. The best practice is only to bump the version when releasing a new version.